MessageViewController/MessageAutocompleteController.swift

```diff
@@ -46,8 +46,17 @@ public final class MessageAutocompleteController: MessageTextViewListener {
     private var typingTextAttributes: [NSAttributedStringKey: Any] {
         var attributes = defaultTextAttributes
         attributes[NSAttributedAutocompleteKey] = false
+        attributes[.paragraphStyle] = paragraphStyle
         return attributes
     }
+    
+    /// The NSAttributedStringKey.paragraphStyle value applied to attributed strings
+    private let paragraphStyle: NSMutableParagraphStyle = {
```

Comment:
Sorry to chime in after. Do we want to return it as a non-mutable `NSParagraphyStyle`?

Reply:
I guess that would make more sense, yeah